src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="info" strict="true" name="XMLConfigTest"
+               packages="org.apache.logging.log4j.test">
```

Comment:
@mhemker is this packages here needed? I think its breaking logging for main app.

Reply:
@vvitvits I don't think you need that entire attribute.